packages/cli/lib/ui.js

```diff
@@ -65,16 +70,22 @@ const uiLink = (linkText, url) => {
 };
 
 /**
- * Returns formatted account name and ID
+ * Returns formatted account name, account type (if applicable), and ID
  *
  * @param {number} accountId
+ * @param {boolean} bold
  * @returns {string}
  */
-const uiAccountDescription = accountId => {
+const uiAccountDescription = (accountId, bold = true) => {
   const account = getAccountConfig(accountId);
-  return chalk.bold(
-    account.name ? `${account.name} (${account.portalId})` : account.portalId
-  );
+  const message = `${account.name} ${
```

Comment:
Not a blocker, but sometimes the nested ternary expressions can be tricky to follow. This would probably also be ok as a few if statements.